src/test/java/strategy/BalanceOperationTest.java

```diff
@@ -0,0 +1,47 @@
+package strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import db.Storage;
+import model.FruitTransaction;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationTest {
+    private static final String APPLE = "apple";
+    private static final Integer AMOUNT_TWO = 20;
+    private static final String BANANA = "banana";
+    private static final Integer AMOUNT_APPLE = 10;
+    private static final Integer AMOUNT_THREE = 15;
+    private static BalanceOperation balanceOperation;
+
+    @Before
+    public void setUp() {
+        balanceOperation = new BalanceOperation();
+    }
+
+    @Before
+    public void cleanStorage() {
+        Storage.clearStorage();
+    }
+
+    @Test
+    public void testDoOperation_AddsTransactionQuantityToStorage() {
```

Comment:
Minor, but I believe `test` prefix in method names is not needed here and in other operation tests